TKGame/Enemies/Goblin/Components/C_Goblin_Physics.cs

```diff
@@ -5,19 +5,20 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
-using TKGame.Components.Concrete;
 using TKGame.Components.Interface;
 using TKGame.BackEnd;
 using Myra.Graphics2D.UI;
 using System.Drawing;
+using TKGame.Players;
 
-namespace TKGame.Components.Concrete
+namespace TKGame.Enemies.Goblin.Components
 {
-    class GoblinEnemy_PhysicsComponent : PhysicsComponent
+    class C_Goblin_Physics : IPhysicsComponent
```

Comment:
C_Goblin_Physics is short for Concrete_Goblin_Physics, right?

Reply:
Yeah you can think of it like that. In my head its "Component_x_y" similar to I for Interface but it has the same meaning as Concrete.